frontend/pages/404.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Link from "next/link";
+import { Button } from "react-bootstrap";
+import classes from "../styles/error.module.css";
+const hello = () => {
+  return (
+    <>
+      <div className={classes.imageBox}>
+        <img
+          className={classes.image}
+          src="https://i.ibb.co/G39013c/img-3.png"
```

Comment:
I think the image has small dimension, on computer screen it looks kinda blurred, please you a different image or reduce the dimensions.

also first download the image into the `/frontend/public/images` folder and then use it, instead of directly using it from the web,